React/Modules/RCTUIManager.h

```diff
@@ -97,7 +97,6 @@ RCT_EXTERN NSString *const RCTUIManagerRootViewKey;
  * @param reactTag the component tag
  * @param completion the completion block that will hand over the rootView, if any.
  *
- * @return the rootView
```

Comment:
That change fixes an actual doc error -- the method does not in fact return anything.